src/renderer/renderer.ts

```diff
@@ -57,6 +58,10 @@ declare const canvas2pdf: any;
 // Some global functions and variables which are only accessible within VSCode:
 declare const vscode: any | null;
 
+type SDFGElementType = 'states' | 'nodes' | 'edges' | 'isedges';
+// If type is explicitly set, dagre typecheck fails with integer node ids
+export type SDFGListType = any[];//{ [key: number]: DagreSDFG };
```

Comment:
Hm, sad that this can't be further typed. But something to deal with this in the overhault.

Reply:
Yes